cherrypy/process/servers.py

```diff
@@ -185,7 +185,7 @@ def _get_base(self):
         if not self.httpserver:
             return ''
         host, port = self.bind_addr
-        if getattr(self.httpserver, 'ssl_adapter', None):
+        if getattr(self,'ssl_certificate', None):
```

Comment:
I've restarted that build to check whether it's a temporary failure. Still, you need to add test for this case checking that what was failing before is fixed now.